src/rdo.rs

```diff
@@ -43,6 +44,41 @@ pub struct RDOPartitionOutput {
 }
 
 // Sum of Squared Error for a wxh block
+fn cdef_dist_wxh_8x8(src1: &PlaneSlice, src2: &PlaneSlice) -> u64 {
```

Comment:
I guess it's called like this because it's the distortion metric used in cdef, but here it's confusing since it's used for everything.